UnitsNet/UnitDefinitions/Mass.json

```diff
@@ -115,6 +115,34 @@
           "Abbreviations": [ "st" ]
         }
       ]
+    },
+    {
+      "SingularName": "ShortHundredweight",
+      "PluralName": "ShortHundredweight",
+      "FromUnitToBaseFunc": "x/0.022046226218487758",
+      "FromBaseToUnitFunc": "x*0.022046226218487758",
+      "XmlDocSummary": "The short hundredweight (abbreviation cwt) is a unit of mass equal to 100 pounds in US and Canada. In British English, the short hundredweight is referred to as the \"cental\".",
+      "XmlDocRemarks": "https://en.wikipedia.org/wiki/Hundredweight",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "cwt" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "LongHundredweight",
+      "PluralName": "LongHundredweight",
+      "FromUnitToBaseFunc": "x/0.01968413055222121",
+      "FromBaseToUnitFunc": "x*0.01968413055222121",
+      "XmlDocSummary": "The long or imperial hundredweight (abbreviation cwt) is a unit of mass equal to 112 pounds in US and Canada.",
+      "XmlDocRemarks": "https://en.wikipedia.org/wiki/Hundredweight",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "cwt" ]
```

Comment:
Confirmed the UnitsNet.AmbiguousUnitParseException:

`UnitsNet.AmbiguousUnitParseException : Cannot parse 'cwt' since it could be either of these: 8, 16
   at UnitsNet.UnitSystem.Parse(String unitAbbreviation, Type unitType) in F:\git\UnitsNet\UnitsNet\CustomCode\UnitSystem.cs:line 227
   at UnitsNet.UnitSystem.Parse[TUnit](String unitAbbreviation) in F:\git\UnitsNet\UnitsNet\CustomCode\UnitSystem.cs:line 200
   at UnitsNet.Mass.ParseUnit(String str, IFormatProvider formatProvider) in F:\git\UnitsNet\UnitsNet\GeneratedCode\UnitClasses\Mass.g.cs:line 1171
   at UnitsNet.Mass.<>c.<Parse>b__117_0(String value, String unit, IFormatProvider formatProvider2) in F:\git\UnitsNet\UnitsNet\GeneratedCode\UnitClasses\Mass.g.cs:line 1086
   at UnitsNet.UnitParser.ParseWithRegex[TUnit](String regexString, String str, ParseUnit`1 parseUnit, IFormatProvider formatProvider) in F:\git\UnitsNet\UnitsNet\CustomCode\UnitParser.cs:line 109
   at UnitsNet.UnitParser.ParseUnit[TUnit](String str, IFormatProvider formatProvider, ParseUnit`1 parseUnit, Func`3 add) in F:\git\UnitsNet\UnitsNet\CustomCode\UnitParser.cs:line 65
   at UnitsNet.Mass.Parse(String str, IFormatProvider culture) in F:\git\UnitsNet\UnitsNet\GeneratedCode\UnitClasses\Mass.g.cs:line 1082
   at UnitsNet.Mass.Parse(String str) in F:\git\UnitsNet\UnitsNet\GeneratedCode\UnitClasses\Mass.g.cs:line 1047
   at UnitsNet.Tests.CustomCode.MassTests.ParseHundredWeightString() in F:\git\UnitsNet\UnitsNet.Tests\CustomCode\MassTests.cs:line 107`